### PR TITLE
Updated the content under Sending Log Events section

### PR DESCRIPTION
**Pull request by fulyauluturk**

### CODE REVIEW

File: docs/images/logs/discovered-logs.png
Comment (otisg) [inline docs/images/logs/discovered-logs.png:1]: Should look more or less the same after F&D1/3 is out?

File: docs/images/logs/sematext-discovery.png
Comment (otisg) [inline docs/images/logs/sematext-discovery.png:1]: Will be slightly different after F&D is out later this week, I believe

File: docs/logs/authorizing-ips-for-syslog.md [hunk 1]
```diff
@@ -13,28 +13,22 @@ logs originating from some IP should be routed.  As such, a given IP
...
-your Logs Management applications), or from the application itself.
+your Sematext account.
 
 ## Method 1: From the Application Settings
 
-First, you have to get to your [list of Logs Management Apps](https://apps.sematext.com/ui/logs). You can do
-that from anywhere by going to **Integrations-\> Apps**.
-
-From the list of applications, click on the **Actions** button next to
+Get to your Logs Apps from the left menu panel. From the list of applications, click on the **Actions** button next to
```
Comment (otisg) [inline docs/logs/authorizing-ips-for-syslog.md:20]: applications ==> Apps

File: docs/logs/authorizing-ips-for-syslog.md [hunk 2]
```diff
@@ -13,28 +13,22 @@ logs originating from some IP should be routed.  As such, a given IP
...
 the Logs Management application you need to work with:
```
Comment (otisg) [inline docs/logs/authorizing-ips-for-syslog.md:21]: Logs Management application ==> Logs App

File: docs/logs/authorizing-ips-for-syslog.md [hunk 3]
```diff
@@ -13,28 +13,22 @@ logs originating from some IP should be routed.  As such, a given IP
...
 
-<img src="/docs/images/logs/syslog-ip-authorize-method-1-min.gif" alt="whitelist syslog IP">
+<img src="/docs/images/logs/logs-app-syslog-ip-authorize-method-1-min.gif" alt="whitelist syslog IP">
 
 ## Method 2: From the Logs Management Application
 
-You can also authorize IPs without leaving the Sematext Logs Management application. To
-do that, click on the **App actions→App Settings** button on the
-upper-right side.
+You can also authorize IPs without leaving the Sematext Logs Management App. To
```
Comment (otisg) [inline docs/logs/authorizing-ips-for-syslog.md:29]: Logs Management App ==> Logs App

File: docs/logs/sending-docker-logs.md [hunk 4]
```diff
@@ -1,50 +1,21 @@
...
+Click on the **"Set Up"** button next to the services from which you want to ship logs. This enables automatic log shipping for the selected services. 
 
-```
+Note that Sematext Agent recognizes log formats from various applications and provides out-of-the-box Dashboards and [Alert Rules](https://sematext.com/docs/guide/alerts-guide/) for each [integration](https://sematext.com/docs/integration/) type. So when you are creating a Logs App, select the integration type based on the services you are running on your container. 
 
-A few seconds later, Nginx logs will be displayed in Sematext: beautiful, structured web server logs, including container metadata.
+<img src="/docs/images/logs/integrations.png" alt="docker environment">
 
-*One second! How does Logagent know  the container log format?*
+You also have the option to create a Generic logs App and create custom Reports based on your log source.
```
Comment (otisg) [inline docs/logs/sending-docker-logs.md:20]: Generic logs App ==> generic Logs App ?

**Review by otisg — APPROVED** [review]
I could do only a shallow review of cosmetic changes.  How it all looks and feels I'll be able to see only after the update.

File: docs/images/logs/discovered-logs.png
Comment (fulyauluturk) [reply docs/images/logs/discovered-logs.png:1]: Correct

**Pull request merged**